VaticanBAV/et/BAVet170.xml

```diff
@@ -0,0 +1,180 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BAVet170" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Anaphora of Our Lady Mary by Cyriacus of Behnesa, Introduction to the Miracles of Mary, two malkǝʾ hymns</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0003BAV"/>
+                        <collection>Aethiopici</collection>
+                        <idno>Aeth. 170</idno>
+                    </msIdentifier>
+
+<msContents>
+<msItem xml:id="ms_i1">
+<locus from="2" to="30"/>
+<title xml:lang="en" ref="LIT1099Anapho">Anaphora of Our Lady Mary by Cyriacus of Behnesa</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="30v" to="41"/>
+<title type="complete" xml:lang="en" ref="LIT1968Mashaf">Introduction to the Miracles of Mary</title>
+<incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="omitted"/>ዝንቱ፡ መጽሐፍ፡ ዘይትነበብ፡ እምቅድመ፡ ያንብቡ፡ ተአም<choice><corr>ሪ</corr><sic>ረ</sic></choice>ሃ፡ ለእግዝእትነ፡ ማርያም፡ ዝንቱ፡ መጽሐፈ፡ ሥርዓት፡ ዘወጽአ፡ እመንበረ፡ ማርቆስ፡ ሐዋርያ፡ እመካነ፡ <ref type="place" corresp="LOC4885Mualla">መሐልቃ፡</ref> ዘምስር፡ ዘአንበሩ፡ ሊቃነ፡ ጳጳሳት፡ ክቡራን፡ መገብያነ፡
+ምሥጢር፡ ርቱዓነ፡ ሃይማኖት፡ ዘትትናገሮሙ፡ እግዝእትነ፡ ማርያም፡ ዘልፈ፡ ቦ፡ አመ፡ ትትናገሮሙ፡ በራዕይ፡ ወቦ፡ አመ፡ ትትናገሮሙ፡ በሕልም፡ ወቦ፡ አመ፡ ትትናገሮሙ፡ በገሃድ፡ እሉ፡ እሙንቱ፡ አባ፡ አብርሃም፡ ወአባ፡
+ወአባ፡ <sic>ማቲዎን፡</sic> ወአባ፡ ማርቆስ፡ ገባርያነ፡ መንክራት፡ ወዜና፡ ነገሮሙሰ፡ ሀሎ፡ ጽሑፍ፡ ውስተ፡ መጽሐፈ፡ ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ዘከመ፡ ተናገረቶሙ፡ ወዘከመ፡ ገብረት፡ ሎሙ፡
```

Comment:
Seems to be a haplology. If it is indeed twice ወአባ፡ in the manuscript, then it should have a sic as well.